opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/netflow5-troubleshooting.adoc

```diff
@@ -0,0 +1,44 @@
+
+[[telemetryd-netflow5-troubleshooting]]
+===== NetFlow v5 Troubleshooting
+
+.Verify in Karaf Console if the `flows` feature is installed and started
+[source]
+----
+list | grep -i fail
+----
+
+The `Geocoder Nomatim` should be listed as `Failure` which is ok.
+
+.Test connectivity to ElasticSearch with curl with a valid response (use -u <elasticUser>) if authentication is enabled
```

Comment:
I'd rephrase this as:
Test the connectivity to ElasticSearch using curl

And add a tip bellow with:
use -u <elasticUser> if authentication is enabled